rpm/python3-pytz.spec

```diff
@@ -0,0 +1,34 @@
+Summary:       Pytz
+Name:          pytz
+Version:       2022.1
+Release:       1
+License:       MIT
+Source0:       %{name}-%{version}.tar.gz
+URL:           https://pypi.org/project/pytz/
```

Comment:
Extra slash here

Reply:
Actually it should point to https://github.com/sailfishos/python3-pytz here